docs/essence.rst

```diff
@@ -956,19 +956,35 @@ Partition operators
 ~~~~~~~~~~~~~~~~~~~
 
 +-------------------------+----------------------------------------------------+
-| ``apart``               | test if two elements are in different parts of     |
-|                         | the partition                                      |
+| ``apart``               | test if a list of elements are not all contained   |
+|                         | in one part of the partition                       |
 +-------------------------+----------------------------------------------------+
 | ``participants``        | union of all parts of a partition                  |
 +-------------------------+----------------------------------------------------+
 | ``party``               | part of partition that contains specified element  |
 +-------------------------+----------------------------------------------------+
 | ``parts``               | partition to its set of parts                      |
 +-------------------------+----------------------------------------------------+
-| ``together``            | test if two elements are in the same part of the   |
-|                         | partition                                          |
+| ``together``            | test if a list of elements are all in the same     |
+|                         | part of the partition                              |
 +-------------------------+----------------------------------------------------+
 
+Examples:
+
+.. code-block:: essence
+
+   letting P be partition({1,2},{3},{4,5,6})
+   find a : bool such that a = apart({3,5},P) /\ !together({1,2,5},P) $ true
+   find b : set of int(1..6) such that b = participants(P) $ {1,2,3,4,5,6}
+   find c : set of int(1..6) such that c = party(4,P) $ {4,5,6}
+   find d : bool such that d = ({{1,2},{3},{4,5,6}} = parts(P)) $ true
+   find e : bool such that e = (together({1,7},P) \/ apart({1,7},P)) $ true
+
+Note that the original Essence definition requires that ``apart({1,7},P)`` and ``together({1,7},P)`` should both be false.
+In contrast, currently the relationship
+  ``apart(L,P) = !together(L,P)``
```

Comment:
I think this is a bug.

Reply:
I think both interpretations are reasonable (and have documented the current behaviour).